src/test/java/picard/fingerprint/FingerprintCheckerTest.java

```diff
@@ -476,4 +480,108 @@ public void testWriteFingerprint() throws IOException {
 
         VcfTestUtils.assertVcfFilesAreEqual(vcfOutput, vcfExpected);
     }
+
+    @DataProvider(name = "identifyContaminantTestDataProvider")
+    public Object[][] identifyContaminantTestDataProvider() {
+        final Path NA12878_fingerprint = Paths.get(TEST_DATA_DIR.getAbsolutePath(), "NA12878_chr1_fingerprint.vcf");
```

Comment:
To convert file to path you can use the following:
```suggestion
        final Path NA12878_fingerprint = TEST_DATA_DIR.toPath().resolve("NA12878_chr1_fingerprint.vcf");
```

Reply:
👍